### PR TITLE
Add similar item warning for add and edit feature

### LINKED ISSUE #134: Warning for possible duplicate items

### PR DESCRIPTION
**Pull request by Md-Fazil**
1. Modify add such that a similar item warning is given when user adds an item with the same name and location in different cases.
2. Modify edit such that a similar item warning is given when users edits an item to the same name and location as an existing item but in different cases.

Closes #134.

### CODE REVIEW

File: src/main/java/seedu/storemando/model/item/Item.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public boolean isExpired() {
     }
 
     /**
-     * Returns true if both items have the same name.
+     * Returns true if both items have the exactly the same name and location.
```
Comment (kumsssss) [inline src/main/java/seedu/storemando/model/item/Item.java]: can add same expiry date also just to make it clearer.

**Review by JayChenYJ — COMMENTED** [review]
LGTM

**Review by kumsssss — APPROVED** [review]
LGTM!

File: src/main/java/seedu/storemando/model/item/Item.java (hunk 1 above)
Comment (Md-Fazil) [reply src/main/java/seedu/storemando/model/item/Item.java]: Fixed.

**Pull request merged**